### PR TITLE
Compile on JDK 11, Use Jakarta EE 10, MicroProfile 5 and Payara 6.2022.1

### PR DESCRIPTION
**Pull request by JamesHillyard**
Updates Payara Examples to use JDK 11, Jakarta EE 10, MicroProfile 5.0 and Payara 6.2022.1. These samples haven't been tested - that will likely be a much larger issue, however they do compile. There are a few known failing tests, these will need to be addressed later.

### CODE REVIEW

File: cloud-providers/azure/template/scripts/setup_payara.sh [hunk 1]
```diff
@@ -16,7 +56,7 @@ function install_packages
 function download_payara
 {
     cd $HOME_DIR
-    rm -rf payara5
+    rm -rf payara6
     wget http://central.maven.org/maven2/fish/payara/distributions/payara/5.192/payara-5.192.zip
```
Comment (aubi) [inline cloud-providers/azure/template/scripts/setup_payara.sh:60]: This probably won't work. For P6, it's https://repo.maven.apache.org/maven2/fish/payara/distributions/payara/6.2022.1/payara-6.2022.1.zip

File: cloud-providers/aws/beanstalk/payara-server-docker-aws-eb/pom.xml [hunk 2]
```diff
@@ -1,4 +1,44 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
+  DO NOT ALTER OR REMOTE COPYRIGHT NOTICES OR THIS HEADER.
```
Comment (aubi) [inline cloud-providers/aws/beanstalk/payara-server-docker-aws-eb/pom.xml:3]: REMOTE COPYRIGHTs are everywhere :-)

**Review by aubi — CHANGES_REQUESTED** [review]
Otherwise it looks good.

File: cloud-providers/aws/beanstalk/payara-server-docker-aws-eb/pom.xml (hunk 2 above)
Comment (JamesHillyard) [reply cloud-providers/aws/beanstalk/payara-server-docker-aws-eb/pom.xml:3]: The one line I didn't copy and paste into the copyright template... 😞 - Will get this fixed

**Review by aubi — APPROVED** [review]

**Pull request merged**